events-api/src/modules/tickets/tickets.repository.ts

```diff
@@ -0,0 +1,52 @@
+import { TicketsCreateDto } from "./dto/requests/tickets-create-dto";
+import { db } from "#/database/database";
+import { v4 } from "uuid";
+import { TicketsModel } from "./tickets.model";
+
+export class TicketsRepository {
+  static async create(newTicket: TicketsCreateDto): Promise<void> {
+    const { eventId, ticketPrice, ticketQuantity } = newTicket;
+
+    await db.execute<TicketsModel>(
+      `
+        INSERT INTO tickets (ticket_id, ticket_quantity, ticket_price, event_id) 
+        VALUES (?, ?, ?, ?)
+      `,
+      [v4(), ticketQuantity, ticketPrice, eventId]
+    );
```

Comment:
Return the created ticket instead of returning nothing

Reply:
![image](https://github.com/user-attachments/assets/323b9d56-00ab-4507-9ca0-d2bc5ee31d9b)

Reply:
Casing is incorrect, use camelCase instead of snake_case

Reply:
done 
![image](https://github.com/user-attachments/assets/8e889c71-4afe-47e2-b4c9-24bc71074077)

Reply:
Here you returning an array of records, please return a single record